app/workers/notifications/mention_worker.rb

```diff
@@ -0,0 +1,11 @@
+module Notifications
```

Comment:
I created a notifications module to mimick the notifications module in the job directory